### DISCUSSION TITLE
Find users by historical name

### DISCUSSION THREAD

**Discussion by GGXXBB** [vrcx-team/VRCX#295] [Ideas]
A user has changed his name. Can I look him up using his historical name, even if I'm not his friend?

**Comment by Natsumi-sama** [comment]
No. But you can use direct access if you know their username/userId, or become friends and give them a note as a nickname.

**Reply by GGXXBB** [reply]
Is there any way to check the Friend Requests I sent before？

**Reply by Natsumi-sama** [reply]
Only if they never accepted it, click refresh in notifications.

**Reply by GGXXBB** [reply]
Thank you for your reply.

**Reply by Natsumi-sama** [reply]
Friend log might have the sent friend request.
